kfdefs/overlays/moc/smaug/opf-jupyterhub/kfdef.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: jupyterhub/jupyterhub
+      name: jupyterhub
+    - kustomizeConfig:
```

Comment:
same as before, can you update this to also have overlays/parameters as it existed in override you removed: 
```
      overlays: []
      parameters:
      - name: notebook_destination
        value: opf-jupyterhub
      - name: s3_endpoint_url
        value: s3.odh.com
```